packages/core/src/Utils/ExchangeCurrencyRate/exchange_currency_rate.js

```diff
@@ -0,0 +1,7 @@
+import BinarySocket from '_common/base/socket_base';
+
+export const getExchangeRate = async (from_currency, to_currency) => {
```

Comment:
I am using the exchange_rates API according to the suggestion that is made by BE in the card.